charts/camunda-platform/templates/connectors/configmap.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "connectors.fullname" . }}
```

Comment:
It needs `-configuration` prefix in the name to make it clear that configmap is just for configuration.